spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -27,6 +27,12 @@ describe('GA4 click tracker', function () {
     spyOn(GOVUK.analyticsGa4.core.trackFunctions, 'getProtocol').and.returnValue('https:')
   })
 
+  afterAll(function () {
+    element = document.createElement('a')
+    element.setAttribute('href', '#')
+    element.click()
+  })
```

Comment:
If you're trying to reset everything in the URL after the `#` you can do `window.history.replaceState(null, null, '#')`

Reply:
Nice, thanks :+1: